adrs/0153-checkout-v2.md

```diff
@@ -179,6 +183,33 @@ A better solution is:
 
 Given a source file path, walk up the directories until the first `.git/config` is found. Check if it matches the self repo (`url = https://github.com/OWNER/REPO`). If not, drop the source file path.
 
+
+### User name and email
```

Comment:
unclear how this will be impacted by [submodules support](https://github.com/actions/checkout/pull/157). probably need to add to local git config for each submodule too